packages/teams-js/src/public/barCode.ts

```diff
@@ -94,6 +94,7 @@ export namespace barCode {
    * @beta
    */
   export function isSupported(): boolean {
```

Comment:
>isSupported [](http://example.com/codeflow?start=18&length=11)

Should the docs for all these `isSupported` methods be updated to indicate that they [throw](https://typedoc.org/tags/throws/) when called before initialization is completed?

Reply:
done